### PR TITLE
{bio} [goolf-1.4.10] RAxML (review)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by pescobar** [comment]
replacing this one https://github.com/hpcugent/easybuild-easyconfigs/pull/1182

File: easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-avx.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'RAxML'
+version = '8.1.1'
+versionsuffix ='-mpi-avx'
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-avx.eb]: no possibility to let it just use whatever the cpu supports?

Comment (boegel) [inline easybuild/easyconfigs/r/RAxML/RAxML-8.1.1-goolf-1.4.10-mpi-avx.eb]: I don't think so, see the Makefile being used in `buildopts`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4beb7de879ecf1c79606 for a full test report.

**Comment by boegel** [comment]
@pescobar: looks good to go, but I need confirmation from Jenkins first to merge...

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**